pyvrp/cpp/search/Route.h

```diff
@@ -112,7 +118,7 @@ class Route
      */
     void update();
 
-    Route(ProblemData const &data);
+    Route(ProblemData const &data, size_t const idx, size_t const typeIdx);
```

Comment:
RouteType was a mistake (sorry I should have checked that better), `idx` should now only be used for `routeIdx` (do you want to call that `route` as well? that seems unrelated to this PR). I replaced `typeIdx` by `vehType` (since otherwise many lines will exceed the line length), which I hope is clear.